openedx/core/djangoapps/edx_global_analytics/tests/test_cache_instance_data.py

```diff
@@ -0,0 +1,100 @@
+"""
+Tests for edX global analytics application cache functionality.
+"""
+
+from datetime import date, datetime
+
```

Comment:
It is second Python, mock is not standard library.
Mock is  standard library from version `3.3`.